src/Pages/List/Components/Drink.js

```diff
@@ -0,0 +1,189 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Tooltip from './Tooltip';
+
+const Drink = ({ toggleTooltip, value, selectedTooltip, sendToServer }) => {
+  const [drink, setDrink] = useState([]);
+  const [isChecked, setIsChecked] = useState({
+    맥주: false,
+    위스키: false,
+    와인: false,
+    칵테일: false,
+  });
+
+  const changeOption = item => {
+    setIsChecked({ ...isChecked, [item.name]: item.checked });
+    const copyDrink = [...drink];
+    copyDrink.includes(item.name)
+      ? setDrink(copyDrink.filter(drink => drink !== item.name))
+      : setDrink([...drink, item.name]);
+  };
+
+  const deleteData = () => {
+    setDrink([]);
+    setIsChecked({
+      맥주: false,
+      위스키: false,
+      와인: false,
+      칵테일: false,
+    });

```

Comment:
```suggestion
const initialCheckedStatus = {
      맥주: false,
      위스키: false,
      와인: false,
      칵테일: false,
    }
    setIsChecked(initialCheckedStatus);
```
로 해주고 state 초기값도 `useState(initialCheckedStatus)` 로 해주면 종류가 늘거나 줄 때마다 두번씩 수정하는 번거로움이 없겠네요

Reply:
대뱍!

Reply:
Drink component의 state가 선언된 곳에서 initialCheckedStatus  를 쓸려면 더 상단에 선언되어야 하는 것 맞죠? import 밑, Drink component  위에 선언하는게 맞을까요?